packages/components/src/interfaces/complex/Accordion.ts

```diff
@@ -10,7 +11,7 @@ export const AccordionSchema = Type.Object(
       title: "Accordion summary",
       description: "The summary for the accordion",
     }),
-    details: Type.Ref(ProseSchema),
+    details: InnerProseSchema,
```

Comment:
nit: maybe can rename as `BaseProseSchema`? That's kinda what we did for Paragraph so wanted to be consistent in the naming

Reply:
sgtm, idk how to name it so i just gave it as `Inner` LULW